web/model/deployment_pb.js

```diff
@@ -13,7 +13,7 @@
 
 var jspb = require('google-protobuf');
 var goog = jspb;
-var global = (function() {
+var global = (function () {
```

Comment:
could you run `yarn lint:fix` to recheck this kind of indent 👀 Sorry if you already did 🙏

Reply:
Those files are generated code so let me fix them in another PR.
This PR should only be for the renaming of the directory.

Reply:
Ah I got your point. Not sure why it was modified. Let me regenerate the code.

Reply:
Addressed by https://github.com/pipe-cd/pipecd/pull/3376/commits/34f28eed158e8805636244aa564ec266ee7ace59